data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,184 @@ mission "Coalition: Outskirts Gauger Ready"
+			`	"Okay," says Ann, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and take me to the Ring of Friendship once you're ready. I'll meet you in the spaceport."`
+
+mission "Sativa: Struggling Author 2"
+	name "Help Ann Register"
+	description "Take Ann Oryza Sativa to the <destination> so she can register with the Heliarchs."
+	source "Ki Patek Ka"
+	destination "Ring of Friendship"
+	to offer
+		has "Sativa: Struggling Author 1: done"
+	on offer
+		conversation
+			`You walk through the spaceport, looking for Ann. She is at a restaurant, but has ordered nothing with a disgusted look on her face. "These aliens! They vomit on their food and slurp all the soup! Disgusting!", she mutters.`
+			choice
+				`	"That's just how they eat."`
+				`	"My ship still has a decent amount of food you can eat."`
+				`	"There should be another restaurant."`
+			`	"<first>! It's you again!", says Ann in an excited tone. "Is the ship ready? Remember, we need to go to the <destination> so I can get registered with the Heliarchs.`
+			choice
+				`	"It's all ready now. Come."`
+			`	Ann stands up from her chair and begins following you to the ship. She settles into her bunk as you prepare for a trip to the <destination>.`
+				accept
+	on enter "Quaru"
+		conversation
+			`As you enter <system>, Ann walks out of her bunk and finds a spot on the side of the bridge, to peer into space. "What a massive ringworld!", she exclaims.`
+			choice
+				`	"It's enormous. They actually stole it from the Quarg, who they rebelled against roughly six thousand years ago."`
+				`	"I'm used to it."`
+			`	"Awesome! This is going to be a great setting and certainly a source of inspiration for my next novel!", says Ann, her voice still in an enthusiastic and excited tone. "This must be where I'm supposed to register, right?"`
+			`	"Yes, this is the <planet>. Prepare your identity documents."`
+	on complete
+		conversation
+			`	Ann is still in awe as you touch down on the ringworld. As the two of you open the hatch, you see yet another Heliarch agent in front of you.`
+			`	"Greetings, humans.  For you, what can I do today?", asks the Heliarch.`
+			`	"This is my friend Ann. I'm getting her registered with the Heliarchs today.", you answer.`
+			`	"To reach the registration office, this way you must go.", replies the Heliarch, pointing to a hallway.`
+			`	You and Ann walk down the hallway as directed, until you spot a registration office. Ann walks into the registration office while you sit down on a nearby bench. After 15 minutes, Ann walks out and approaches you. "They finally registered me.". I'm off to the spaceport now to eat something. Meet me there so we can discuss our next move.`
+
+mission "Sativa: Struggling Author 3"
+	name "Back to <planet>"
+	description "Now that Ann Oryza Sativa has completed the registration process, buy her a translation device."
```

Comment:
So the player doesn't get a translation device at any point currently, and cannot buy them either. We know civilians can get them, but we don't have an explicit "this is where you buy them" shop mentioned.
Both of the PR'd Intros for the Coalition campaigns gift the player with a translation device when either of them is completed (either become a Heliarch or join the Lunarium).

It's something of a given that Coalition folks that are better off can afford translation devices, and since we know Ann hasn't been doing so well on her career, it'd be odd if it was expected of her to get a translation device and communicate with the publisher like that. The publisher would have their translators though, and even if we assume it'd be a lesser known publishing company, you could get Heliarchs to translate for the interactions initially, before the book is a success and lets the parties afford the translators.

I think it'd be best to instead, for this third mission, focus on how Ann pitches the book to the Kimek publishers, or maybe even have this mission set stopovers to the three species capitals (Saros, Ki Patek Ka, Ahr), so that it's more her trying to pitch to the different species and see which one likes it best (the mission'd go away once landing on Ki Patek Ka, getting replaced by a `landing` mission that'd be the next step on the chain).
On the idea of the Heliarchs mediating it, even if you opt for the publisher having translators, you could have Heliarch agents come in and watch the negotiation between Ann and the Kimek, like a watchful force there to make sure no anti-Heliarchy ideas are coming from this foreign author (this could also factor in with the timer for the book to be published, something like the Heliarchs inspecting the content a thousand times over to make absolutely sure it's not going against them).

Reply:
I've decided on the name `House Torimek` for the publishing company.

For the timer, if First Last is in Coalition space, they get a message from the publishers. But if First Last leaves, then there's no way for the publishers to get in contact, so First Last will get a message reminding them to.

Maybe a mission like:
> As you go through your diary, you recall having helped an author named Ann Oryza Sativa publish her book in Coalition space. Maybe you should go and see how it's going?


Maybe I'll just make mission three about asking some Heliarchs for help? And then they send a Heliarch over to help First Last and Ann with the book publishing work?

Reply:
> Maybe a mission like:
> 
> > As you go through your diary, you recall having helped an author named Ann Oryza Sativa publish her book in Coalition space. Maybe you should go and see how it's going?

I feel like saying First Last has a diary is too much assumption. They could just remember about the author.

Reply:
We have a captain's log, but telling the player that they check the log when they might not even know about it could be strange.